packages/atomic/src/components/search/facets/atomic-popover/atomic-popover.tsx

```diff
@@ -0,0 +1,174 @@
+import {
+  buildSearchStatus,
+  SearchStatus,
+  SearchStatusState,
+} from '@coveo/headless';
+import {Component, h, Listen, State, Element} from '@stencil/core';
+import {
+  BindStateToController,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../../utils/initialization-utils';
+import {Button} from '../../../common/button';
+import {Hidden} from '../../../common/hidden';
+import {Bindings} from '../../atomic-search-interface/atomic-search-interface';
+import ArrowBottomIcon from 'coveo-styleguide/resources/icons/svg/arrow-bottom-rounded.svg';
+import {InitPopoverEventPayload} from './popover-event';
+
+/**
+ * @internal
+ * The `atomic-popover` displays a popover menu view of any facet
+ * @part value-button - The button used to display the popover menu
+ * @part value-label - The popover value label
+ * @part arrow-icon - The arrow icon
+ * @part placeholder - The placeholder displayed when the facet isn't ready
+ * @part popover-wrapper - The wrapper that contains the value-button and the slot-wrapper
+ * @part slot-wrapper - The wrapper that contains the facet/slot
```

Comment:
```suggestion
 * @part slot-wrapper - The wrapper that contains the 'facet' or 'slot'.
```
What's a slot in this context? First time it's mentioned in this file 🤔

Reply:
A slot is a placeholder that can be filled with another component